src/main/java/org/sonarlint/cli/report/SourceProvider.java

```diff
@@ -34,18 +32,19 @@
 
 public class SourceProvider {
   private final Logger logger = Logger.get();
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Make the "logger" logger private static final and rename it to comply with the format "LOG(?:GER)?". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=squid%3AS1312)